src/components/screens/auth/components/RegisterButton.tsx

```diff
@@ -73,7 +73,7 @@ const RegisterButton: React.FC<Props> = props => {
 
     try {
       const profile = await OneWelcomeSdk.registerUser(providerId, ['read']);
-      CurrentUser.id = profile.id;
+      CurrentUser.id = profile.userProfile.id;
```

Comment:
why is there a userprofile within a profile? i think the profile variable name is not accurate anymore